lib/supabase/in_process_artists/getArtists.ts

```diff
@@ -0,0 +1,19 @@
+import { supabase } from "@/lib/supabase/client";
+
+// Fetch artists by query (case-insensitive, partial)
+export async function getArtists(query: string, limit = 10) {
+  if (!query || query.trim().length === 0) {
+    return null;
+  }
+  if (query.length > 100) {
+    return null;
+  }
```

Comment:
Will this endpoint fail if the search query is longer than 100 characters?
What negative effects happen if I send a query with more than 100 characters?
- If none, please remove this check 🚮